cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx

```diff
@@ -0,0 +1,150 @@
+    }
+];
+
+ClarityIcons.addIcons(flaskIcon, computerIcon, cpuIcon, memoryIcon);
+
+function ClusterTopologyStep(props: Partial<StepProps>) {
+    const {handleValueChange, currentStep, goToStep, submitForm} = props;
+    const {state, dispatch} = useContext(WcStore);
+    const methods = useForm<ClusterTopologyStepFormInputs>({
+        resolver: yupResolver(clusterTopologyStepFormSchema),
+    });
+    const {
+        register,
+        handleSubmit,
+        setValue,
+        formState: {errors},
+    } = methods;
+
+    const onSubmit: SubmitHandler<ClusterTopologyStepFormInputs> = (data) => {
+        if (Object.keys(errors).length === 0) {
+            if (goToStep && currentStep && submitForm && handleValueChange) {
+                handleValueChange('WORKLOAD_CLUSTER_NAME', data.WORKLOAD_CLUSTER_NAME, currentStep, errors);
+                handleValueChange('SELECTED_WORKER_NODE_INSTANCE_TYPE', data.SELECTED_WORKER_NODE_INSTANCE_TYPE, currentStep, errors);
+                goToStep(currentStep + 1);
+                submitForm(currentStep);
+            }
+        }
+    };
+
+    return (<div className="wizard-content-container" key="cluster-topology">
+        <p cds-text="heading">Workload Topology Settings</p>
+        <br/>
+
+        {ManagementClusterInfoBanner(state.data.SELECTED_MANAGEMENT_CLUSTER)}
+        <br/>
+        <div cds-layout="grid gap:md" key="section-holder">
+            <div cds-layout="col:6" key="cluster-name-section"> {ClusterNameSection(errors, register)} </div>
+            <div cds-layout="col:6" key="instance-type-section"> {WorkerNodeInstanceTypeSection(errors, register)} </div>
+        </div>
+        <br/>
+        <CdsButton onClick={handleSubmit(onSubmit)}>NEXT</CdsButton>
+    </div>);
+}
+
+function WorkerNodeInstanceTypeSection(errors: any, register: any) {
+    return <div cds-layout="vertical gap:lg gap@md:lg col@sm:6 col:6">
+        <div cds-layout="cols:6">Select a worker node instance type</div>
+        <div cds-layout="grid gap:md align:fill">
+            {
+                workerNodeInstanceTypes.map(instanceType => {
+                    return InstanceTypeInList(instanceType, register);
+                })
+            }
+        </div>
+        { errors.SELECTED_WORKER_NODE_INSTANCE_TYPE &&
+            <CdsControlMessage status="error">{errors.SELECTED_WORKER_NODE_INSTANCE_TYPE.message}</CdsControlMessage>
+        }
+    </div>;
+}
+
+function InstanceTypeInList(instance: WorkerNodeInstanceType, register: any) {
+    return <>
+        <div className="text-white" cds-layout="col:1"><CdsIcon shape={instance.icon}></CdsIcon></div>
+        <input className="inputradio" cds-layout="col:1" type="radio" value={instance.id}
+               {...register("SELECTED_WORKER_NODE_INSTANCE_TYPE")} />
+        <div className="text-white" cds-layout="col:10">{instance.name} {instance.description}</div>
+    </>
+        ;
```

Comment:
nitpick, move ; to end of previous line

Reply:
An issue of line length, enforced by eslint.